src/components/Button/Button.tsx

```diff
@@ -17,3 +20,10 @@ export const Button = (props: ButtonProps) => {
 
   return <MuiButton className={classes.button} {...props} />;
 };
+
+const defaultButtonProps: ButtonProps = {
+  rel: 'noopener noreferrer',
+  target: '_blank',
```

Comment:
Wouldn't it render all of our buttons as links?
Also, I know it will be more robust, but in my opinion, it would be better if we pass those props explicitly when using the Button component

Reply:
I suppose it would, please double-check @dominikslawkowski